Lib/idlelib/help.py

```diff
@@ -160,26 +160,33 @@ def __init__(self, parent, filename):
         Text.__init__(self, parent, wrap='word', highlightthickness=0,
                       padx=5, borderwidth=0, width=uwide, height=uhigh)
 
-        normalfont = self.findfont(['TkDefaultFont', 'arial', 'helvetica'])
-        fixedfont = self.findfont(['TkFixedFont', 'monaco', 'courier'])
-        self['font'] = (normalfont, 12)
-        self.tag_configure('em', font=(normalfont, 12, 'italic'))
-        self.tag_configure('h1', font=(normalfont, 20, 'bold'))
-        self.tag_configure('h2', font=(normalfont, 18, 'bold'))
-        self.tag_configure('h3', font=(normalfont, 15, 'bold'))
-        self.tag_configure('pre', font=(fixedfont, 12), background='#f6f6ff')
-        self.tag_configure('preblock', font=(fixedfont, 10), lmargin1=25,
-                borderwidth=1, relief='solid', background='#eeffcc')
-        self.tag_configure('l1', lmargin1=25, lmargin2=25)
-        self.tag_configure('l2', lmargin1=50, lmargin2=50)
-        self.tag_configure('l3', lmargin1=75, lmargin2=75)
-        self.tag_configure('l4', lmargin1=100, lmargin2=100)
+        self.create_fonts()
+        self.configure_tags()
 
         self.parser = HelpParser(self)
         with open(filename, encoding='utf-8') as f:
             contents = f.read()
         self.parser.feed(contents)
+
+        self.bind_events()
         self['state'] = 'disabled'
+        self.focus_set()
+
+    def create_fonts(self):
+        "Create fonts to be used with tags."
+        self.base_size = idleConf.GetOption('main', 'EditorWindow',
+                                            'font-size', type='int')
+        normalfont = self.findfont(['TkDefaultFont', 'arial', 'helvetica'])
+        fixedfont = self.findfont(['TkFixedFont', 'monaco', 'courier'])
+
+        self.fonts = fonts = {}
```

Comment:
I don't really like the doubling of the configuration code, but I have not thought of anything better yet.

Reply:
I didn't make any changes for this.